tools/nl/embeddings/sheets/embeddings_us_filtered.csv

```diff
@@ -1242,6 +1242,55 @@ Number,Name,Id,Description Strings,PaLM Generated Alternatives,ChatGPT descripti
-,,dc/topic/WetBulbTemperature,,,for this area what is the temperature at which the human body can't cool itself anymore; what is the wet bulb temperature; how high or low is the wet bulb temperature predicted to be; give me the degree of the wetbulb temperature; wet bulb temperature prediction range,Wet Bulb Temperature,,for this area what is the temperature at which the human body can't cool itself anymore; what is the wet bulb temperature; how high or low is the wet bulb temperature predicted to be; give me the degree of the wetbulb temperature; wet bulb temperature prediction range;Wet Bulb Temperature;,
-,,dc/topic/Temperature,,,what's the temperature like; what's the climate here; tell me about the weather;,Temperature,,what's the temperature like; what's the climate here; tell me about the weather;;Temperature;,
-,,dc/235cmp60ywm46,number of law industry workers; people working in law; how many legal service workers are there; population working in legal services; population of legal service workers,,,Population Working in Legal Services,,number of law industry workers; people working in law; how many legal service workers are there; population working in legal services; population of legal service workers;Population Working in Legal Services;,
+,,dc/topic/WetBulbTemperature,,,for this area what is the temperature at which the human body can't cool itself anymore; what is the wet bulb temperature; how high or low is the wet bulb temperature predicted to be; give me the degree of the wetbulb temperature; wet bulb temperature prediction range,Wet Bulb Temperature,,for this area what is the temperature at which the human body can't cool itself anymore; what is the wet bulb temperature; how high or low is the wet bulb temperature predicted to be; give me the degree of the wetbulb temperature; wet bulb temperature prediction range;Wet Bulb Temperature;,6
+,,dc/topic/Temperature,,,what's the temperature like; what's the climate here; tell me about the weather;,Temperature,,what's the temperature like; what's the climate here; tell me about the weather;;Temperature;,5
+,,dc/235cmp60ywm46,number of law industry workers; people working in law; how many legal service workers are there; population working in legal services; population of legal service workers,,,Population Working in Legal Services,,number of law industry workers; people working in law; how many legal service workers are there; population working in legal services; population of legal service workers;Population Working in Legal Services;,6
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
+,,,,,,,,,0
```

Comment:
Why these empties?

Reply:
I don't know. Likely some rows added in the Sheets at the end by mistake? I have gone ahead and force deleted those rows at the end but I don't know why this happened.